index.js

```diff
@@ -33,4 +46,6 @@ function getRuleURI(ruleId) {
   };
 }
 
+getRuleURI.plugins = plugins;
```

Comment:
I’ve exported it in case a module wants to use it to get the raw URLs. Should I remove the export? I could also extract the parsing logic into a separate file.